blackmarket/Extensions/ViewExtension.swift

```diff
@@ -122,6 +122,49 @@ extension UIView {
       constant: offset
     ).isActive = true
   }
+  
+  enum SeparatorPosition {
+    case top
+    case bottom
+    case left
+    case right
+  }
+
+  @discardableResult
+  func addSeparator(at position: SeparatorPosition, color: UIColor, weight: CGFloat = 1.0 / UIScreen.main.scale, insets: UIEdgeInsets = .zero) -> UIView {
+    let view = UIView()
+    view.backgroundColor = color
+    view.translatesAutoresizingMaskIntoConstraints = false
+    self.addSubview(view)
```

Comment:
Avoid expliciy use of `self` whenever possible.
```suggestion
    addSubview(view)
```